packages/alfa-aria/src/dom/predicate/is-programmatically-hidden.ts

```diff
@@ -34,39 +34,32 @@ export function isProgrammaticallyHidden(
   );
 }
 
-const cache = Cache.empty<Device, Cache<Context, Cache<Node, boolean>>>();
-
-function hasHiddenAncestors(
+function _hasHiddenAncestors(
```

Comment:
This is just a demo of using the new decorator, not directly related to the changes.